examples/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains an example configuration file, as well as necessary files to run a demo of Transition. This demo corresponds to the territory of the Réseau de transport de Longueuil area.
```

Comment:
maybe add the URL for the RTL, to give more contexte to people (https://www.rtl-longueuil.qc.ca/)